src/util/constants.ts

```diff
@@ -1,14 +1,17 @@
 import { v4 as uuidv4 } from 'uuid';
 
-export const totalName = '전체';
+export const totalFolderName = '전체';
+export const totalFolderId = uuidv4();
 
 export const updateBtnsId = {
   share: uuidv4(),
   change: uuidv4(),
   delete: uuidv4(),
 };

```

Comment:
```suggestion
export const updateBtnsId = {
  share: uuidv4(),
  change: uuidv4(),
  delete: uuidv4(),
} as const;

```

이렇게 활용하면 readonly 값으로 표기하는 것이기 때문에 보다 좁혀진 타입을 제공받을 수 있습니다.